packages/database/src/migrations/20210615034653-AddCovidFijiSubDivisonDashboardGroup-modifies-data.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+import { insertObject, arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+const dashboardGroup =
+  // Covid Fiji
+  {
+    organisationLevel: 'SubDistrict',
+    userGroup: 'COVID-19',
+    organisationUnitCode: 'FJ',
+    dashboardReports: '{FJ_Covid_Fiji_District_COVID-19_Num_Of_1st_Vaccine_Dose_Taken,FJ_Covid_Fiji_District_COVID-19_Num_Of_2nd_Vaccine_Dose_Taken,FJ_Total_Number_Of_People_1st_Dose_Covid_Vaccine,FJ_Total_Number_Of_People_2nd_Dose_Covid_Vaccine}',
```

Comment:
I'm checking with Juliana if we need to also move other dashboards to SubDistrict level as well. But if so, we might have to check if `FJ_Covid_Fiji_District_COVID-19_Num_Of_1st_Vaccine_Dose_Taken` and `FJ_Covid_Fiji_District_COVID-19_Num_Of_2nd_Vaccine_Dose_Taken` still work at `SubDistrict` level because these are implemented only for `District` level

Reply:
Ok I've just checked. They're working fine at `sub_district` level even tho `aggregationEntityType` is `district`. All good!

Reply:
Ah so may need to copy as a sub_district report?

Reply:
There's really no sub_district reports to copy unfortunately. I'm actually surprised that they work as I've checked. But I think the ideal solution is still cloning the district ones to sub_district and remove `aggregationEntityType`: `district`. I guess I'm just a bit scared because I don't really understand how the district ones could also work in sub_district